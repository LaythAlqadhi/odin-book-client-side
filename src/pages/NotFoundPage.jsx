import React from 'react';

function NotFoundPage() {
  return (
    <div className="flex h-full flex-col items-center justify-center">
      <h1 className="text-4xl font-bold">404 - Not Found</h1>
      <p className="text-gray-400">
        Oops! The page you are looking for doesn&apos;t exist.
      </p>
    </div>
  );
}

export default NotFoundPage;
