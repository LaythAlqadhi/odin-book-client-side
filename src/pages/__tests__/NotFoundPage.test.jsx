import React from 'react';
import { render, screen } from '@testing-library/react';
import NotFoundPage from '../NotFoundPage';

describe('NotFoundPage component', () => {
  it('renders NotFoundPage component correctly', () => {
    render(<NotFoundPage />);

    expect(screen.container).toMatchSnapshot();
  });
});
