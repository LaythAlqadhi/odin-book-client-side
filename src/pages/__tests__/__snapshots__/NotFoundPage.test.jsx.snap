// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFoundPage component > renders NotFoundPage component correctly 1`] = `undefined`;
